src/Events/QR/Routes.php

```diff
@@ -75,7 +76,7 @@ public function do_register(): void {
 		 */
 		$this->route_prefix = apply_filters( 'tec_events_qr_route_prefix', 'qr' );
 
-		$this->salt = substr( wp_salt( Settings::get_qr_slug() ), 0, 8 );
+		$this->salt = substr( wp_salt( QR_Controller::QR_SLUG ), 0, 8 );
```

Comment:
Suggestion, create a method that returns the salt. but also allows someone to filter it.

What that accomplishes is; Until someone asks for the salt we dont waste processing power generating it.